src/backend/distributed/executor/multi_router_executor.c

```diff
@@ -284,7 +285,7 @@ ExecuteDistributedModify(Task *task)
 		}
 		else
 		{
-			ereport(WARNING, (errmsg("modified %d tuples, but expected to modify %d",
+			ereport(WARNING, (errmsg("modified %ld tuples, but expected to modify %ld",
```

Comment:
If we do go ahead with this review, the `%ld` should be `INT64_FORMAT` and not a literal `%ld`.